contentcuration/contentcuration/frontend/channelEdit/move/MoveModal.vue

```diff
@@ -270,17 +266,14 @@
         });
       },
       moveToClipboard() {
-        this.moveContentNodesToClipboard(this.moveNodeIds).then(() => {
+        // Clipboard only holds references to copies, so we cannot "move" a node to the clipboard
+        this.copyAll({ id__in: this.moveNodeIds }).then(() => {
```

Comment:
I wonder if it makes sense to keep this option around in the first place since I would expect the move modal would only handle moves. If it makes sense to keep it here though, could you update the button string to match the action?

Reply:
I can update the string. When I brought it up last, it didn't seem like there was a decision, so I left it.